Error with IR Analysis on MSSQL

When executing the IR analysis, the following exception is thrown:

```
Caused by: java.sql.BatchUpdateException: Invalid object name 'cteCohortCombos'.
```

If a `studyWindow` is specified, the execution fails due to the lack of temp table qualifier in likes 125, 126 of IRAnalysisQueryBuilder.java.

--- a/src/main/java/org/ohdsi/webapi/ircalc/IRAnalysisQueryBuilder.java
+++ b/src/main/java/org/ohdsi/webapi/ircalc/IRAnalysisQueryBuilder.java
@@ -122,8 +122,8 @@ public class IRAnalysisQueryBuilder {
               String.format("UNION\nselect combos.target_id, combos.outcome_id, t.subject_id, t.cohort_start_date, %s as followup_end, 0 as is_case",
               SqlUtils.dateStringToSql(analysisExpression.studyWindow.endDate))
       );
-      endDatesQuery.append("\nFROM cteCohortCombos combos");
-      endDatesQuery.append("\nJOIN  cteCohortData t on combos.target_id = t.target_id and combos.outcome_id = t.outcome_id");
+      endDatesQuery.append("\nFROM #cteCohortCombos combos");
+      endDatesQuery.append("\nJOIN  #cteCohortData t on combos.target_id = t.target_id and combos.outcome_id = t.outcome_id");
 
       resultSql = StringUtils.replace(resultSql, "@EndDateUnions", endDatesQuery.toString());
     }

--- a/src/main/java/org/ohdsi/webapi/ircalc/IRAnalysisTasklet.java
+++ b/src/main/java/org/ohdsi/webapi/ircalc/IRAnalysisTasklet.java
@@ -16,6 +16,8 @@
 package org.ohdsi.webapi.ircalc;
 
 import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.common.collect.ImmutableList;
+import com.odysseusinc.arachne.commons.types.DBMSType;
 import org.ohdsi.sql.SqlSplit;
 import org.ohdsi.sql.SqlTranslate;
 import org.ohdsi.webapi.common.generation.CancelableTasklet;
@@ -98,6 +100,8 @@ public class IRAnalysisTasklet extends CancelableTasklet {
       }
       
       String expressionSql = analysisQueryBuilder.buildAnalysisQuery(analysis, options);
+      
+
       String translatedSql = SqlTranslate.translateSql(expressionSql, source.getSourceDialect(), sessionId, oracleTempSchema);
       return SqlSplit.splitSql(translatedSql);
     } catch (Exception e) {

